recipes/ogg/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ogg" %}
+{% set version = "1.3.2" %}
+{% set sha256 = "e19ee34711d7af328cb26287f4137e70630e7261b17cbe3cd41011d73a654692" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  lib{{ name }}-{{ version }}.tar.gz
+  url: http://downloads.xiph.org/releases/{{ name }}/lib{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - pkgconfig  # [unix]
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+about:
+  home: https://www.xiph.org/
+  license: Public Domain
```

Comment:
Right, they say BSD-like license on the website, but I'll just keep it to BSD 3-clause then.